src/main/java/com/eviive/personalapi/controller/SkillController.java

```diff
@@ -72,7 +63,11 @@ public ResponseEntity<Slice<SkillDTO>> findAll(
         summary = "Create a skill",
         responses = {
             @ApiResponse(responseCode = "201", description = "Created"),
-            @ApiResponse(responseCode = "400", description = "Bad Request")
+            @ApiResponse(
+                responseCode = "400",
+                description = "Bad Request",
+                content = @Content(schema = @Schema(implementation = ProblemDetail.class))
+            )
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'annotation array initialization' child has incorrect indentation level 12, expected level should be 16.